modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/discovery/impl/DiscoveryServiceImpl.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.mdm.mobileservices.windows.services.discovery.impl;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.mdm.mobileservices.windows.common.Constants;
+import org.wso2.carbon.mdm.mobileservices.windows.services.discovery.beans.DiscoveryRequest;
+import org.wso2.carbon.mdm.mobileservices.windows.services.discovery.DiscoveryService;
+import org.wso2.carbon.mdm.mobileservices.windows.services.discovery.beans.DiscoveryResponse;
+
+import javax.jws.WebService;
+import javax.ws.rs.core.Response;
+import javax.xml.ws.BindingType;
+import javax.xml.ws.Holder;
+import javax.xml.ws.soap.Addressing;
+import javax.xml.ws.soap.SOAPBinding;
+
+/**
+ * Implementation class of Discovery Request. This class implements the first two services
+ * of device enrolment stage.
+ */
+@WebService(endpointInterface = Constants.DISCOVERY_SERVICE_ENDPOINT, targetNamespace = Constants
+		.DISCOVERY_SERVICE_TARGET_NAMESPACE)
+@Addressing(enabled = true, required = true)
+@BindingType(value = SOAPBinding.SOAP12HTTP_BINDING)
+public class DiscoveryServiceImpl implements DiscoveryService {
+
+	private static Log logger = LogFactory.getLog(DiscoveryServiceImpl.class);
```

Comment:
We generally use "log" to name Logger instances. Even though there's no any specific rule for this, what's being used generally across the platform is "log". So, let's stick to that.